### ISSUE TITLE
Firefox 87 to limit the referrer for all cross-origin requests, does this addon still provide extra benefits?

### ISSUE THREAD

**Issue by rugabunda** [claustromaniac/poop#31]
Does this have anything to do with your addon? Does this make your addon less necessary or redundant?

https://www.ghacks.net/2021/03/22/firefox-87-to-limit-the-referrer-for-all-cross-origin-requests/

**Comment by crssi** [comment]
No... the main point of this extension is to strip out the `origin` attribute.
The stripping out `referer` attribute is an addendum to make it less obvious when `origin` is stripped out.

Cheers
